### PR TITLE
Add Elasticsearch Dashboard docs

### PR DESCRIPTION
**Pull request by DRoyyy**
Signed-off-by: DRoyyy <diptaroy@appscode.com>

### CODE REVIEW

**Review by raihankhan — DISMISSED** [review]
LGTM!

File: docs/guides/elasticsearch/concepts/elasticsearch-dashboard/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+KubeDB allows providing a template for database pod through `spec.podTemplate`. KubeDB operator will pass the information provided in `spec.podTemplate` to the StatefulSet created for the Elasticsearch database.
+
+KubeDB accepts the following fields to set in `spec.podTemplate`:
+
+- metadata
+  - annotations (pod’s annotation)
+
+- controller
+  - annotations (statefulset’s annotation)
```
Comment (kamolhasan) [inline docs/guides/elasticsearch/concepts/elasticsearch-dashboard/index.md]: deployment's annotation

File: docs/guides/elasticsearch/concepts/elasticsearch-dashboard/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+- `username`: Must be `elastic` for `x-pack`, and `admin` for `OpenSearch`.
+- `password`: Password for the `elastic`/`admin` user.
+  If `spec.authSecret` is not set, dashboard operator will use the authSecret from referred database object.
+
+### spec.configSecret
+
+`spec.configSecret` is an optional field that allows users to provide custom configuration for `ElasticsearchDashboard`. It contains a k8s secret name that holds the configuration files for `ElasticsearchDashboard`. If not provided, operator generated configurations will be applied to dashboard. If `configSecret` is provided, it will be merged with the operator-generated configuration. The user-provided configuration has higher precedence over the operator-generated configuration. The configuration file names are used as secret keys.
+
+#### Kibana:
```
Comment (kamolhasan) [inline docs/guides/elasticsearch/concepts/elasticsearch-dashboard/index.md]: Make them list items

**Review by kamolhasan — APPROVED** [review]
LGTM!

**Pull request merged**